rapid_pro_tools/rapid_pro_client.py

```diff
@@ -22,12 +37,44 @@ def get_flow_id(self, flow_name):
         return matching_flows[0].uuid
 
     def get_flow_ids(self, flow_names):
+        """
+        Gets the ids for a list of flow names.
+
+        :param flow_names: Names of the flows to retrieve the ids of.
+        :type flow_names: list of str
+        :return: The Rapid Pro ids for the given flow names.
+        :rtype: list of str
+        """
         return [self.get_flow_id(name) for name in flow_names]
 
     def get_flow_definitions_for_flow_ids(self, flow_ids):
+        """
+        Gets the definitions for the flows with the requested ids from Rapid Pro.
+
+        :param flow_ids: Ids of the flows to export the definitions of.
+        :type flow_ids: list of str
+        :return: An export object containing all of the requested flows, their dependencies, and triggers.
+        :rtype: temba_client.v2.types.Export
+        """
         return self.rapid_pro.get_definitions(flows=flow_ids, dependencies="all")
 
-    def get_raw_runs_for_flow_id(self, flow_id, range_start_inclusive=None, range_end_exclusive=None):
+    def get_raw_runs_for_flow_id(self, flow_id, range_start_inclusive=None, range_end_exclusive=None,
+                                 raw_export_log=None):
+        """
+        Gets the raw runs for the given flow_id from RapidPro.
+        
+        :param flow_id: Id of the flow to download the runs of.
+        :type flow_id: str
+        :param range_start_inclusive: Start of the date-range to download runs from. If set, only downloads runs
+                                      last modified since that date, otherwise downloads from the beginning of time.
+        :type range_start_inclusive: datetime.pyi | None
+        :param range_end_exclusive: End of the date-range to download runs from. If set, only downloads runs
+                                    last modified before that date, otherwise downloads until the end of time.
+        :param raw_export_log: File to write the raw data downloaded during this function call to.
```

Comment:
consider: raw_export_log_file

And in the description add a note that it's going to be jsony.

Reply:
changed